package.json

```diff
@@ -64,6 +65,8 @@
     "clean": "scripts/run-script --parallel clean",
     "dev": "scripts/run-script --parallel dev",
     "dev-test": "jest --bail --watch \"^(?!.*\\.integ\\.spec\\.js$)\"",
+    "docs:dev": "vuepress dev docs",
+    "docs:build": "vuepress build docs",
```

Comment:
Why not create `docs/package.json` directly?

```json
    "docs:dev": "vuepress dev .",
    "docs:build": "vuepress build .",
```

Reply:
I'm following this: https://vuepress.vuejs.org/guide/getting-started.html#inside-an-existing-project

And the way it worked for XCP-ng doc. I'd like to avoid "taking risks" on using something else, except if you tell me exactly how to do it.